lib/rubocop/ast/node/mixin/method_identifier_predicates.rb

```diff
@@ -1,22 +1,69 @@
 # frozen_string_literal: true
 
+require 'set'
```

Comment:
The gem `parser` (and maybe others) already `require 'set'`. I'd remove it entirely, but if you prefer having it for clarity, it belongs in `lib/reubocop/ast.rb`.

Reply:
> but if you prefer having it for clarity

yes, this. 
And imo it is better to not depend on something else required it.
Moved to suggested location.